openedx/features/pakx/lms/pakx_admin_app/constants.py

```diff
@@ -0,0 +1 @@
+Group_Training_Managers = "Training Manager"
```

Comment:
constants should be defined in all capital letters with underscores separating words. 
please follow pep8

Reply:
Done!